templates/user/auth/reset_passwd.tmpl

```diff
@@ -30,10 +30,31 @@
 							</div>
 						</div>
 						{{end}}
+						{{if .has_two_factor}}
+						<h4 class="ui dividing header">
```

Comment:
can you indent the text?

Reply:
The rest of the stuff in that template does not indent.

Reply:
@zeripath why no start with this ...?

Reply:
we probably need to think more carefully about this indentation in templates. 

If you indent the here - the html output from the template is malindented.